docker-compose.ac.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright © 2020-2022 Intel Corporation. All rights reserved.
```

Comment:
I am good with this compose file snippet in this PR, but not the full EdgeX one which goes beyond scope of this PR.